src/libs/ReportUtils.ts

```diff
@@ -6856,6 +6865,33 @@ function buildOptimisticUnHoldReportAction(created = DateUtils.getDBTime()): Opt
     };
 }
 
+function buildOptimisticRetractedReportAction(created = DateUtils.getDBTime()): OptimisticRetractedReportAction {
+    return {
+        reportActionID: rand64(),
+        actionName: CONST.REPORT.ACTIONS.TYPE.RETRACTED,
+        actorAccountID: currentUserAccountID,
+        pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD,
+        message: [
+            {
+                type: CONST.REPORT.MESSAGE.TYPE.COMMENT,
+                text: 'retracted',
+                html: `<muted-text>retracted</muted-text>`,
```

Comment:
Should this use `getRetractedMessage` and be a `RETRACTED` type? I'm guessing you're just copying what gets sent from the BE but let me know if this needs an update.

Reply:
no, this is not used to display.

 so im just matching the backend

we show the reportaction based on the reportAction type